gooddata-java-model/pom.xml

```diff
@@ -92,6 +92,10 @@
             <artifactId>spring-web</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-web</artifactId>
```

Comment:
We don't really want Spring dependency in model module - we want it to have as few dependencies as possible.
Please move constants using `UriTemplate` into Service the same way as in other services ;)

Reply:
Returned to its original state.